gpiocontrol/source/c/src/kaa_demo.c

```diff
@@ -76,7 +75,7 @@ static void kaa_GPIOToggle_info_request(void *context
 
     gpio_led[event->gpio->id] = event->gpio->status;
 
-    int gpio_id = event->gpio->id >= NUM_GPIO_LEDS?NUM_GPIO_LEDS:event->gpio->id;
+    int gpio_id = event->gpio->id >= (NUM_GPIO_LEDS ? NUM_GPIO_LEDS : event->gpio->id);
```

Comment:
This changes semantics. The previous expression was parsed as:
```c
int gpio_id = (event->gpio->id >= NUM_GPIO_LEDS) ? NUM_GPIO_LEDS : event->gpio->id;
```

That's what I meant by "around condition".